src/nifgen/examples/nifgen_standard_function.py

```diff
@@ -2,22 +2,22 @@
 
 import argparse
 import nifgen
+import time
 
 supported_waveforms = list(nifgen.Waveform.__members__.keys())[:-1]  # no support for user-defined waveforms in example
 parser = argparse.ArgumentParser(description='Generates the standard function.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-parser.add_argument('-n', '--name', default='PXI1Slot2', help='Resource name of a National Instruments Function Generator')
+parser.add_argument('-n', '--resource_name', default='PXI1Slot2', help='Resource name of a National Instruments Function Generator')
 parser.add_argument('-w', '--waveform', default=supported_waveforms[0], choices=supported_waveforms, type=str.upper, help='Standard waveform')
 parser.add_argument('-f', '--frequency', default=1000, type=float, help='Frequency (Hz)')
 parser.add_argument('-a', '--amplitude', default=1.0, type=float, help='Amplitude (Vpk-pk)')
 parser.add_argument('-o', '--offset', default=0.0, type=float, help='DC Offset (V)')
 parser.add_argument('-p', '--phase', default=0.0, type=float, help='Start Phase (deg)')
+parser.add_argument('-t', '--time', default=5, type=float, help='Generation Time')
```

Comment:
if type is float, then default should be typed as float: 5.0
help should not be title case, and it should indicate the units.